cmake/modules/extensions.cmake

```diff
@@ -528,6 +528,11 @@ function(zephyr_library_cc_option)
   endforeach()
 endfunction()
 
+# Similar as above functions support add_dependencies
```

Comment:
minor nit.
Although this function is almost self-explanatory given its name, then I think the help text is not useful and should either be a real help text or completely removed.

Which functions above should I consider when looking above ?
`zephyr_library_compile_options` doesn't seem a good candidate.
(not to mention the term `above` is very sensitive if code is relocated within this file)

Reply:
Thanks, removed comment.